test/unit/DCAStrategies/dca-strategies-management-handler.spec.ts

```diff
@@ -92,11 +101,55 @@ contract('DCAStrategiesManagementHandler', () => {
         expect(strategyId).to.be.equal(1);
         expect(strategyName).to.be.equal(strategy.name);
         expect(owner).to.be.equal(strategy.owner);
-        expect(SHARES.length).to.be.equal(strategy.tokens.length);
-        SHARES.forEach((s: any, i: any) => {
-          expect(tokens[i].share).to.be.equal(s.share);
-          expect(tokens[i].token).to.be.equal(s.token);
-        });
+        compareTokens(SHARES, tokens);
+      });
+    });
+  });
+
+  describe('createStrategy', () => {
+    let owner: Wallet;
+    const NAME = ethers.utils.formatBytes32String('Optimism Ecosystem - v1');
```

Comment:
I think we can share this (and maybe some of the other constants) between all tests